cvx/markowitz/builder.py

```diff
@@ -145,7 +145,7 @@ def build(self):
                 self.constraints[f"{name_model}_{name_constraint}"] = constraint
 
         problem = cp.Problem(self.objective, list(self.constraints.values()))
-        assert problem.is_dpp(), "Problem is not DPP"
+        # assert problem.is_dpp(), "Problem is not DPP"
```

Comment:
Currently rewriting to fix this before we merge.

Reply:
I think we need this check back?

Reply:
Done in [17a3484](https://github.com/cvxgrp/cvxmarkowitz/pull/118/commits/17a3484ef9ae780f9dc67ab2acac1aec2aec06b2)